src/manipulation.js

```diff
@@ -292,13 +292,13 @@ jQuery.extend( {
 						}
 					}
 
-					// Support: Chrome <= 35-45+
+					// Support: Chrome <=35 - 45+
 					// Assign undefined instead of using delete, see Data#remove
 					elem[ dataPriv.expando ] = undefined;
 				}
 				if ( elem[ dataUser.expando ] ) {
 
-					// Support: Chrome <= 35-45+
+					// Support: Chrome <=35 - 45+
```

Comment:
https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69626900 :
> It may turn out that I'm the one to do it, but certainly not before core 3.0.